torchgen/model.py

```diff
@@ -440,7 +446,13 @@ def from_yaml(
 
         funcs = e.pop("func")
         assert isinstance(funcs, str), f"not a str: {funcs}"
-        func = FunctionSchema.parse(funcs)
+
+        namespaced_funcs = funcs.split("::")
```

Comment:
We don't support arbitrary levels of nesting namespaces so you can pass 2 to split